extensions/cache/src/main/java/org/radargun/stages/cache/test/LoadStage.java

```diff
@@ -38,15 +36,15 @@
 
    @Property(doc = "Generator of keys (transforms key ID into key object). Default is 'string'.",
```

Comment:
The doc for the next three properties needs to be updated to reflect the new default before the state is checked